### PR TITLE
PWGGA/GammaConv: added period LHC18b8 for pp 5TeV JJ MC

### PR DESCRIPTION
**Pull request by lleardin**

### CODE REVIEW

File: PWGGA/GammaConv/AliConvEventCuts.h [hunk 1]
```diff
@@ -262,6 +262,7 @@ class AliConvEventCuts : public AliAnalysisCuts {
         kLHC17j7,             //!< MC for Xe-Xe 5.44 TeV HIJING
         kLHC17l3b,            //!< anchored LHC17p/q pass 1 - general purpose w/GEANT3,
         kLHC17l4b,            //!< anchored LHC17p/q pass 1 - general purpose w/GEANT4,
+        kLHC18b8,             //!< anchored LHC17p/q pass 1 - jet-jet MC w/GEANT3,
```
Comment (dmuhlhei) [inline PWGGA/GammaConv/AliConvEventCuts.h]: Could you please increment ClassDef?

Comment (lleardin) [reply PWGGA/GammaConv/AliConvEventCuts.h]: Done

Comment (dmuhlhei) [reply PWGGA/GammaConv/AliConvEventCuts.h]: Thanks :+1:

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**